README.md

```diff
@@ -31,3 +44,51 @@ imap_data_access.download("imap/mag/l0/2024/01/imap_mag_l0_raw_202040101_2024010
 # Upload a calibration file that exists locally
 imap_data_access.upload("imap/mag/calibration/test_calibration.txt")
 ```
+
+## Configuration
+
+### Data directory
```

Comment:
It might be helpful to include an example of this structure, e.g.:

```
The directory structure is as follows:

imap/
  <instrument>/
    <data_level>/
      <year>/
        <month>/
          <filename>

for example:

imap/
  swe/
    l0/
      2024/
        01/
          imap_swe_l0_sci_20240105_20240105_v00-00.pkts
```